### PR TITLE
Doc: Update integration.rst

### PR DESCRIPTION
**Pull request by stifoon**

### CODE REVIEW

File: documentation/navitia.io/source/integration.rst [hunk 1]
```diff
@@ -1978,7 +1978,9 @@ date_time
 +==========================+======================+================================+
 | additionnal_informations | Array of String      | Other information: TODO enum   |
 +--------------------------+----------------------+--------------------------------+
-| date_times               | Array of String      | Date time                      |
+| departure_date_times     | Array of String      | Date time                      |
```
Comment (antoine-de) [inline documentation/navitia.io/source/integration.rst]: isn't it `departure_date_time` with only one date time ?
Actually I don't understand this, didn't you want to change the stop_date_time section to split `date_time  date-time   A date time` into a departure/arrival field ?

**Comment by kinnou02** [comment]
skip ci

File: documentation/navitia.io/source/integration.rst [hunk 2]
```diff
@@ -2058,12 +2064,19 @@ See `interface`_ section.
...
 
-.. _datetime:
+.. _iso-date-time:
 
-datetime
-########
+ISO-date-time
+#############
+
+Navitia manage every date or times as UTC date times. The web-service
```
Comment (pbougue) [inline documentation/navitia.io/source/integration.rst]: manage`s`

**Comment by pbougue** [comment]
Ok for me

**Pull request merged**